README.md

```diff
@@ -1,2 +1,22 @@
-# hw01
-Repository for homework 01
+# **Homework 1**
+## Biography ##
+***Basic Information***  
+
+Name:Ruixi(Ray) Mao  
+
+*  Current Status: third year grad student in Sociology 
+  +  Academic Interest: Economic Sociology; 
+  +  Current Research: Capitalism; Entrepreneurship  
+  
+[here's my linkedin profile]()
+
+***Other Interests***  
+
+Travel: I recently went ot Tibet:
+
+![](IMG_1192.jpg)
```

Comment:
being that your image is in the same folder as the .md file, I'm wondering if the only thing you'd need for this to work is a `/`, so `![](/IMG_1192.jpg)`